docs/source/usersguide/input.rst

```diff
@@ -1192,6 +1234,24 @@ Each ``material`` element can have the following attributes or sub-elements:
 
     *Default*: None
 
+  :macroscopic:
+    The ``macroscopic`` element is similar to the ``nuclide`` element, but,
+    recognizes that some multi-group libraries may be providing material
+    specific macroscopic cross sections instead of always providing nuclide
+    specific data like in the continuous-energy case.  To that end, the
+    macroscopic element has attributes/sub-elements called ``name``, and ``xs``.
+    The ``name`` attribute is the name of the cross-section for a
+    desired nuclide while the ``xs`` attribute is the cross-section
+    identifier. One example would be as follows:
+
+    .. code-block:: xml
+
+        <macroscopic name="UO2" xs="71c" />
```

Comment:
Is there ever a situation where you'd want to specify more than one `<macroscopic>` in the same `<material>`. I'm thinking it might make more sense to have macroscopic as a direct attribute of material, i.e.
```XML
<material id="1" macroscopic="UO2" />
```
Also, perhaps we ought to forgo having "xs" for MG cross sections. It doesn't really add anything, and we will probably eventually move away from specifying "xs" for continuous-energy, so in that sense we may not want to tie ourselves to it too much. What do you think?